### ISSUE TITLE
Alt text is not descriptive [../project-setup files] 

### ISSUE THREAD

**Issue by timngyn** [aws-amplify/docs#7070]
**Describe the content issue**:
Images listed below do not have descriptive/informative alt text. For markdown the alt text is located between the square brackets, `![alt text](...)`.

The list below isn't exhaustive since I found these images based on the word count for the alt text, but it should be a good starting point to make our alt text better for people who use screen readers.

Some resources to help write alt text:
- https://webaim.org/techniques/alttext/
- https://www.a11yproject.com/posts/are-you-making-these-five-mistakes-when-writing-alt-text/

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/project-setup/ios/create-application/10_createProject.mdx:
```
   6  
   7: ![GSA](/images/project-setup/20_1_createProject.png)
   8  

  14  
  15: ![GSA](/images/project-setup/20_2_createProject.png)
  16  
  17  After tapping Next, **select where you would like to save your project**, then **tap Create**.  In this example, you will choose: **~/Developer**
  18: ![GSA](/images/project-setup/20_3_createProject.png)
  19  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/project-setup/ios/create-application/20_install_libraries.mdx:
```
   2  
   3: ![Add package dependency](/images/project-setup/20_4_add-package-dependency.png)
   4  

   8  
   9: ![Search for repo](/images/project-setup/20_5_search-amplify-repo.png)
  10  

  21  
  22: ![Select dependencies](/images/project-setup/20_7_select-dependencies.png)
  23  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib-v1/project-setup/ios/combine/combine.mdx:
```
  101  
  102: ![image](/images/combine-xcode.png)
  103  
```

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib-v1/project-setup/ios/create-application/10_createProject.mdx:
```
   4  
   5: ![GSA](/images/project-setup/20_1_createProject.png)
   6  

  12  
  13: ![GSA](/images/project-setup/20_2_createProject.png)
  14  
  15  After tapping Next, **select where you would like to save your project**, then **tap Create**.  In this example, you will choose: **~/Developer**
  16: ![GSA](/images/project-setup/20_3_createProject.png)
  17  
```

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib-v1/project-setup/ios/create-application/20_install_libraries.mdx:
```
   6  
   7: ![Add package dependency](/images/project-setup/20_4_add-package-dependency.png)
   8  

  12  
  13: ![Search for repo](/images/project-setup/20_5_search-amplify-repo.png)
  14  

  25  
  26: ![Select dependencies](/images/project-setup/20_7_select-dependencies.png)
  27  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib-v1/project-setup/ios/create-application/31_provisionBackend.mdx:
```
   2  
   3: ![GSA](/images/project-setup/50_1_dragDrop.png)
   4  

  10  
  11: ![GSA](/images/project-setup/50_2_addFiles.png)
  12  
```
